rustual-boy-core/src/v810.rs

```diff
@@ -265,6 +295,26 @@ impl V810 {
     pub fn step(&mut self, interconnect: &mut Interconnect) -> (usize, bool) {
         let original_pc = self.reg_pc;
 
+        #[cfg(not(feature = "cpu-trace"))]
+        macro_rules! trace_add {
+            ($($arg:expr),*) => {};
+        }
+
+        #[cfg(feature = "cpu-trace")]
+        macro_rules! trace_add {
+            ($te:expr) => {
+                if self.trace_enabled {
+                    let trace_entry = $te;
```

Comment:
nit: If all this line does is rename the value from the macro, might as well just name the expr `trace_entry`. In the logging stuff (iirc) we had a separate line that would take a _reference_ to the value to make sure that we wouldn't get a dead code warning if the feature was disabled; I don't really see the point here.